topic-operator/src/main/java/io/strimzi/operator/topic/cruisecontrol/CruiseControlClientImpl.java

```diff
@@ -127,10 +126,11 @@ public String topicConfiguration(List<KafkaTopic> kafkaTopics) {
             builder.header("Authorization", buildBasicAuthValue(authUsername, authPassword));
         }
         HttpRequest request = builder.build();
+        LOGGER.traceOp("Request: {}", request);
         
         // send request and handle response
         return httpClient.sendAsync(request, HttpResponse.BodyHandlers.ofString()).thenApply(response -> {
-            LOGGER.traceOp("Response: {}", response);
+            LOGGER.traceOp("Response: {}{}", response, hasBody(response) ? ", body: " + response.body() : "");
```

Comment:
Honestly, I'm not sure I understand what exactly are you trying to do here ...
* What is wrong with the existing log line? Does it not print the body? Or what is the problem with it?
* Why do you need to check the body? Having a simple `null` in the message seems self-explanatory. So why don't you just do:
  ```
  LOGGER.traceOp("Response: {} with body {}", response, response.body());
  ```
  I assume in the worst case it would write `with body null` which is IMHO completely understandable.

(the same applies to the same below)

Reply:
> Does it not print the body?

Exactly.

> I assume in the worst case it would write with body null which is IMHO completely understandable

Ok, we can simplify then.